### DISCUSSION TITLE
[Feature] Internal/external connection profiles

### DISCUSSION THREAD

**Discussion by hussainalhaddad** [immich-app/immich#7667] [Feature Request]
### The feature

Hi

Would it be possible to add a connection profile that switches between internal or external IP address based on the wifi network?

I've seen this implemented in many apps, it really helps with speed and bandwidth saving.

Example attached [nzb360]

Thanks

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by hussainalhaddad** [comment]
![Screenshot_20240306_082835_nzb360](https://github.com/immich-app/immich/assets/55337126/8cb943ab-772d-4a08-acf6-5f2de1357736)

**Comment by bo0tzz** [comment]
Duplicate of #308
